micrometer-core/src/test/java/io/micrometer/core/instrument/binder/cache/HazelcastCacheMetricsTest.java

```diff
@@ -39,7 +53,7 @@ void cacheMetrics() {
         map.put("key", "value");
         map.get("key");
 
-        assertThat(registry.find("cache.gets").value(Statistic.Count, 1.0).meter()).isPresent();
-        assertThat(registry.find("cache.puts").value(Statistic.Count, 1.0).meter()).isPresent();
+        assertThat(registry.find("cache.gets").functionTimer().map(FunctionTimer::count)).hasValue(1L);
+        assertThat(registry.find("cache.puts").functionTimer().map(FunctionTimer::count)).hasValue(1L);
```

Comment:
`FunctionTimer`'s `count` is unusual that is returns a `long` instead of a `double`. I think it is fine that they are different but wanted to point out the difference in case it was unintentional.

Reply:
Thanks for pointing it out. Unlike a normal counter whose units are unknown, the counters inside of timers always count occurrences. Depending on the unit, it isn't unreasonable to expect that a regular counter might increment by a fractional value. Occurrences are always integral, however.